### ISSUE TITLE
UIE 用 evaluate.py 和 taskflow 抽取结果不同

### ISSUE THREAD

**Issue by Shiina18** [PaddlePaddle/PaddleNLP#2428]
paddlenlp 2.3.1, paddlepaddle-gpu 2.3.0.post111
Linux, python 3.6

train.txt (一条)
```json
{"content": "（二）工业和建筑业全年规模以上工业增加值比上年增长11.4%。", "result_list": [], "prompt": "建筑业增加值同比变化百分比"}
```

finetune
```
--num_epochs 1
--seed 0xC7
--valid_steps 1
--model uie-tiny
```

输入训练集的那条数据, 结果 evaluate.py 抽取为空, taskflow 抽取为 '增长11.4%'.

```python
paddlenlp.Taskflow(
        'information_extraction',
        schema=['建筑业增加值同比变化百分比'],
        task_path='./checkpoint/model_best',
        model='uie-tiny'
)
```

evaluate.py 
max_seq_len=512, model='uie-tiny', model_path='./checkpoint/model_best', test_path='./data/train.txt'

finetune.py
dev_path='./data/dev.txt', device='gpu', init_from_ckpt=None, learning_rate=1e-05, logging_steps=10, max_seq_len=512, model='uie-tiny', num_epochs=1, save_dir='./checkpoint', seed=199, train_path='./data/train.txt', valid_steps=1

**Comment by linjieccc** [comment]
您好，有没有可能model_best保存的是用其他训练集微调得到的模型参数文件，训练集只有一条负例（抽取结果为空）的情况下不会保存最佳模型文件
